src/components/ValidationMessage/CorrectValidation.tsx

```diff
@@ -0,0 +1,21 @@
+import CorrectIcon from './CorrectIcon'
+
+type Props = {}
+
+const CorrectValidation = ({}: Props) => {
```

Comment:
```suggestion
const CorrectValidation = () => {
```

If the component has no props, there is no need to declare them